### ISSUE TITLE
Document the requirement for org.clojure/data.xml version

### ISSUE THREAD

**Issue by jwr** [yogthos/clj-rss#20]
I just went through a lengthy debugging session to find out why an old blog generator broke. I was getting a rather bizarre error:

```
   No implementation of method: :gen-event of protocol: #'clojure.data.xml/EventGeneration found for class:
   clojure.lang.PersistentArrayMap
```

As it turns out, the project used the official stable version of org.clojure/data.xml (0.0.8) (for other XML processing), while clj-rss requires a newer (currently "alpha") version and won't work with 0.0.8.

I think it might be a good idea to warn about this in the README, it might save others some time.

**Comment by yogthos** [comment]
I think it might be fine to use version 0.0.8 for data.xml. The library doesn't depend on any specific features from the newer version.

**Comment by jwr** [comment]
I get the exception described above when I use 0.0.8. I think if instead of passing maps to clj-rss you create `clojure.data.xml/element`s, things will work, but maps do not work for me.

**Comment by yogthos** [comment]
Yeah, I just tested locally and can confirm. Looks like it happened [here](https://github.com/yogthos/clj-rss/commit/b14e9ac169a8533d7383eb5ef31ad53504a3cb11). If you'd be up for doing a PR to get it to work with 0.0.8 I could push out a new release.

**Comment by jwr** [comment]
I would, but TBH I have no idea what to do to make it work with 0.0.8. I am completely unfamiliar with how clojure.data.xml works.

I thought a warning in the README would be a temporary "fix". I'm sorry for not being more constructive here…

**Comment by yogthos** [comment]
No worries, I'll add a warning for now but I'll keep it open to see about either switching to 0.0.8 or removing the dependency entirely.
